CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Breaking changes
 
 - [\#300](https://github.com/arkworks-rs/algebra/pull/300) (`ark-ec`) Change the implementation of `Hash` trait of `GroupProjective` to use the affine co-ordinates.
+- [\#306](https://github.com/arkworks-rs/algebra/pull/306) (`ark-ff`) Remove `asm` feature flag.
```

Comment:
```suggestion
```

Reply:
The other CHANGELOG entry also no longer applies. Any suggestion?